src/ResourceManager/ManagedServiceIdentity/Commands.ManagedServiceIdentity/help/AzureRM.ManagedServiceIdentity.md

```diff
@@ -0,0 +1,22 @@
+---
+Module Name: AzureRM.ManagedServiceIdentity
+Module Guid: 2a27df8b-15e3-4fb9-b885-22c6bcafc203
+Download Help Link: {{Please enter FwLink manually}}
+Help Version: {{Please enter version of help manually (X.X.X.X) format}}
+Locale: en-US
+---
+
+# AzureRM.ManagedServiceIdentity Module
+## Description
+{{Manually Enter Description Here}}
```

Comment:
Please add descriptions for each of these cmdlets - if you regenerate the description for the individual modules should be automatically filled in, but you will have to fill the module description manually.